### ISSUE TITLE
Question regarding search result

### ISSUE THREAD

**Issue by seungwooYoo** [arcelien/pba#13]
Hi, 

Firstly, thanks for sharing the codes for your research. Currently I've been trying to reproduce the results using PyTorch. For comparisons, I ran your search codes given scripts for svhn. I found that after finishing the search, there are multiple search policy outcomes. I'm sure that I should select the best validation results policy outcomes. Interestingly, the difference of the validation results from search outcomes are not that high. Can you briefly explain the reasons or some insights ? Second question is it seems that depending on learning schedule, the searched policy could be different. Can you tell some observations or experiences you have ? 

Thanks in advance.

**Comment by arcelien** [comment]
During explore-and-explore, workers with bad validation error get overwritten with weights from good validation error, so at the end of training all workers have likely branched from some small set of workers with good validation error. This would lead to low variance towards the end, especially as learning rate is also approaching 0 due to our cosine schedule.

**Comment by seungwooYoo** [comment]
Thanks for clarification. 👍
